umm-spec-lib/resources/xml-schemas/dif10/UmmCommon_1.2.xsd

```diff
@@ -580,6 +580,8 @@ technical contact
             <xs:enumeration value="Parent Dataset"/>
             <xs:enumeration value="Related Dataset"/>
             <xs:enumeration value="Other"/>
+            <xs:enumeration value="IsPreviousVersionOf"/>
+            <xs:enumeration value="IsNewVersionOf"/>
         </xs:restriction>
```

Comment:
You also need to change the DIF-10 DatasetProgressEnum. This is the same as CMR's CollectionProgress.